wordpress-mobile/WordPress-Android
# Move Story option to the top of the main Create menu

This PR is pretty straightforward and just updates the position of the "Story post" action inside the main Create menu in the My Site screen, from the bottom of the list to the top. 

For more context on this change, check this internal ref: pbArwn-26B-p2#comment-2975

Here's how the change looks like:

| Before | After |
| ------- | ----------- |
|![story-last](https://user-images.githubusercontent.com/830056/111845762-4535b200-88e4-11eb-8460-11c0ebedb77c.png)|![story-first](https://user-images.githubusercontent.com/830056/111845768-4830a280-88e4-11eb-8ce0-2d90821f2167.png)|

### To test

Notes: 

1. Open the app.
1. Make sure you have selected a site that supports stories.
1. On the My Site screen, tap the FAB.
1. Notice that the menu actions have the following order: "Story post", "Blog post" and "Site page".

To test:

## Regression Notes
1. Potential unintended areas of impact
Unlikely, but maybe post and page creation.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual testing.

3. What automated tests I added (or what prevented me from doing so)
None.

PR submission checklist:
